tests/sentry/auth/test_staff.py

```diff
@@ -182,10 +182,11 @@ def test_login_saves_session(self):
         request.user = user
         assert staff.is_active
 
-        data = request.session.get(SESSION_KEY)
+        # See mypy issue: https://github.com/python/mypy/issues/9457
+        data = request.session.get(SESSION_KEY)  # type:ignore[unreachable]
```

Comment:
Annoying bug with mypy not assuming new assert value could change, guido responded to issue tho which is cool
https://github.com/python/mypy/issues/9457